tests_external/skipped_tests_numpy.tbl

```diff
@@ -2881,6 +2881,7 @@ tests/test_umath.py::test_rint_big_int
 tests/test_umath.py::TestRoundingFunctions::test_object_direct
 tests/test_umath.py::TestRoundingFunctions::test_object_indirect
 tests/test_umath.py::test_signaling_nan_exceptions
+tests/test_umath.py::TestSign::test_sign_dtype_nan_object
```

Comment:
what is the reason?
Provide info on description, please.

Reply:
This test failed with error:
DPNP error: in function dpnp_dtype_to_DPNPFuncType() type '<class 'object'>' is not supported

Reply:
`object` is a (global) variable. By default it is bound to a built-in class which is the root of the type hierarchy.

`dpnp_dtype_to_DPNPFuncType` can not convert `object` to `dpnpc` function types (DPNP_FT_DOUBLE, DPNP_FT_FLOAT, DPNP_FT_LONG, ...). 
See this internal func impl [here (https://github.com/IntelPython/dpnp/blob/master/dpnp/dpnp_algo/dpnp_algo.pyx#L256:L271)

You need fallback on `numpy` for case when dtype is object.

Reply:
> You can fallback on numpy for case when dtype is object.

Sorry, my fault. It is bad proposal, just leave it as is.
No need doing fallback on `numpy` for dparray own methods.